kodilla-collections-advanced/src/main/java/com/kodilla/collections/adv/exercises/homework/FlightFinder.java

```diff
@@ -0,0 +1,37 @@
+package com.kodilla.collections.adv.exercises.homework;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FlightFinder {
+
+    private static List<Flight> flights;
+
+    public List<Flight> findFlightsFrom(String departure) {
+        flights = FlightRepository.getFlightTable();
+        List<Flight> flightDeparture = new ArrayList<>();
+        for (Flight flight : flights) {
+            if (flight.departure.equals(departure)) {
+                flightDeparture.add(flight);
+            }
+        }
+        if (flightDeparture.size() == 0) {
```

Comment:
ten if jest niepotrzebny. Jak nic w petli for nie dodamy do listy, to ona bedzie pusta